### PR TITLE
Improve common instructions

### LINKED ISSUE #14: Streamline instructions
Something along the lines of:

```
Click [here] to configure and download resinOS for your new device

Write the OS file you downloaded to an SD card. We recommend using Etcher.

Insert the freshly burnt SD card into the Technologic TS-4900 board.
    
    Warning! This will also completely erase internal storage media, so please make a backup first.

Enable the SD boot jumper present on your specific base board.

Power the device on and wait for it to perform a shut down. Please wait until all LEDs are off.

Remove the resinOS installation SD card.

Disable the SD boot jumper present on your specific base board, then remove and re-connect power to the board.

Your device should appear in your application dashboard within a few minutes. Have fun!

For more in depth information, please refer to our getting started guide.
```

https://www.flowdock.com/app/rulemotion/resin-frontend/threads/iPe7AFmkpqYImeWAdoYyhd40ddU

### PR DESCRIPTION
**Pull request by LucianBuzzo**
Fixes #14

change-type: minor

### CODE REVIEW

File: common/instructions.coffee [hunk 1]
```diff
@@ -87,8 +76,8 @@ module.exports =
 		Unmount your USB key with <code>umount /dev/sddN</code> where N is your USB key number.
 	'''
 
-	EJECT_SD: 'Safely eject the freshly burnt SD card and insert into the <%= TYPE_NAME %>.'
-	EJECT_USB: 'Safely eject the freshly burnt USB key and insert into the <%= TYPE_NAME %>.'
+	EJECT_SD: 'Insert the freshly burnt SD card into the <%= TYPE_NAME %>.'
+	EJECT_USB: 'Insert the freshly burnt USB key into the <%= TYPE_NAME %>.'
```
Comment (thgreasi) [inline common/instructions.coffee]: Based declared name, it seems weird that we don't mention `eject` any more, but it's totally more straightforward :+1:

File: common/instructions.coffee [hunk 2]
```diff
@@ -126,6 +115,6 @@ module.exports =
 
 	BOARD_REPOWER: 'Remove and re-connect power to the board.'
 
-	REMOVE_INSTALL_MEDIA: 'Remove the resin.io installation media.'
+	REMOVE_INSTALL_MEDIA: 'Remove the resinOS installation media.'
```
Comment (thgreasi) [inline common/instructions.coffee]: Nice catch :+1:

File: common/instructions.coffee [hunk 3]
```diff
@@ -35,7 +24,7 @@ module.exports =
 		<br>
 		Make 100% sure it is the correct drive and click “Write”.
 		<br>
-		<strong>Warning!</strong>: Please make sure you have backed-up your SD card first,
+		<strong role="label">Warning!</strong>: Please make sure you have backed-up your SD card first,
```
Comment (thgreasi) [inline common/instructions.coffee]: What's with the `role="label"` that were added on all `<strong>` tags?
Couldn't find it [in the spec](https://www.w3.org/TR/wai-aria-1.1/#x5-4-definition-of-roles) even though I'm not an expert on that.
Would it make sense to use something different, like `alert`
Will we handle them in the UI?

Comment (LucianBuzzo) [reply common/instructions.coffee]: Good idea, lets use `role="alert"`. In the UI we can handle them using a attribute selector.

**Pull request merged**